lumicks/pylake/group.py

```diff
@@ -21,3 +22,25 @@ def __getitem__(self, item):
         else:
             cls = channel_class(thing)
             return cls.from_dataset(thing)
+
+    def keys(self):
+        """Return group names at this level"""
+        return list(self.h5.keys())
+
+    def __iter__(self):
+        self._idx = -1
+        return self
+
+    def __next__(self):
+        """Return key names for the iterator analogously to HDF5"""
+        if self._idx >= len(self.keys()) - 1:
+            raise StopIteration
+
+        self._idx += 1
+        return self.keys()[self._idx]
+
+    def __repr__(self):
+        """Return type name and members of the group"""
+        name = self.__class__.__name__
+        members = ''.join(f"{e}, " for e in self.keys())
```

Comment:
`members = ", ".join(str(k) for k in self.keys())` -- then you don't need to remove the trailing comma with `[:-2]`

Reply:
👍